tests/Geb/positive/Core/test005.jvc

```diff
@@ -1,3 +1,3 @@
 -- basic arithmetic
 
-5 + 2 * 3
+5 + 2 * 3
```

Comment:
Avoid changes that don't really change anything. Revert to the version with the newline.